.github/workflows/fonts.yml

```diff
@@ -0,0 +1,112 @@
+name: Fonts
+
+on:
+  pull_request_target:
+    branches: [ master ]
+    types: [ labeled ] # 'build fonts' label
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'build fonts')
+    outputs:
+      image: ${{ steps.check-image.outputs.result }}
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ format('refs/pull/{0}/merge', github.event.pull_request.number) }}
+        persist-credentials: false # minimize exposure and prevent accidental pushes
+
+    - name: Remove label
+      uses: actions/github-script@v3
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          github.issues.removeLabel({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: "build fonts",
+          });
+
+    - name: Check image
+      id: check-image
+      uses: actions/github-script@v3
+      with:
+        # https://github.community/t/github-token-has-no-access-to-new-container-rest-apis/170395
+        github-token: ${{ secrets.GH_PACKAGES_TOKEN }}
+        result-encoding: string
+        script: |
+          /* eslint-disable camelcase, max-len, no-console */
+          const org = context.repo.owner;
+          const package_name = "fonts";
+          const version = "${{ hashFiles('dockers/fonts/Dockerfile') }}";
+
+          let packages;
+          try {
+              packages = (await github.request('GET /orgs/{org}/packages/{package_type}/{package_name}/versions', {
+                  org,
+                  package_type: "container",
+                  package_name,
+              })).data;
+          } catch (e) {
+              packages = [];
+              console.error(e);
+          }
+
+          core.setOutput("exists", packages.some(p => p.metadata.container.tags.includes(version)));
+          return `ghcr.io/${org}/${package_name}:${version}`.toLowerCase();
+
+    - name: Build and push image
```

Comment:
I thought they would add too much overhead because of `docker/setup-qemu-action` and `docker/setup-buildx-action`, but it seems they are not requried and `buildx` is already installed.